src/main/java/org/opensearch/sdk/sample/helloworld/HelloWorldExtension.java

```diff
@@ -51,6 +53,11 @@ public List<ExtensionRestHandler> getExtensionRestHandlers() {
         return List.of(new RestHelloAction());
     }
 
+    @Override
+    public List<ExtensionsManager.ExtensionInterfaceType> getImplimentedInterfaces() {
+        return List.of(ExtensionInterfaceType.ACTION,ExtensionInterfaceType.CIRCUITBREAKER);
```

Comment:
We don't want to change HelloWorld to this if it doesn't actually implement these extensions.  We can add them to a test class though.

Reply:
Got it. I just added this, just to let you know that values are coming to the other side. Will put it in testcase side

Reply:
I have removed it